TeachingRecordSystem/src/TeachingRecordSystem.Core/DataStore/Postgres/Models/TpsCsvExtractLoadResultType.cs

```diff
@@ -0,0 +1,24 @@
+namespace TeachingRecordSystem.Core.DataStore.Postgres.Models;
+
+[Flags]
+public enum TpsCsvExtractLoadResultType
+{
+    Success = 0,
```

Comment:
Ideally a `[Flags]` enum has a plural name so maybe something like
```suggestion
public enum TpsCsvExtractLoadErrors
{
    None = 0,
```